challenges/objects-arrays.js

```diff
@@ -123,3 +196,35 @@ Stretch: If you haven't already, convert your array method callbacks into arrow
 
 */
 
+
+
+//Let the record show I don't like fat arrow - compression makes the functions and order of operations harder to understand at a glance
+// Plus, some things just break when converted over
+
+console.log(graduates.map(names => names.university).sort());
+
+console.log(graduates.map(nameEmail => `${nameEmail.first_name} ${nameEmail.email}`));
+
+console.log(zooAnimals.map(name => name.animal_name.toLowerCase()));
+
+
+// Not sure why this isn't working.  Another reason I don't like fat arrows - forEach doesn't really work with return statements, and fat arrow doesn't like .push
+// For some reason I don't understand, forEach only works when I newArray.push.
+//console.log(zooAnimals.forEach(scien => `Name: ${scien.animal_name}, Scientific: ${scien.scientific_name}`));
+
+
+// Not sure why this isn't working
+//console.log(graduates.includes("Uni").map(name => name.university));
```

Comment:
Your order is backwards here. It should look like this.

```js
graduates.filter(graduate => graduate.university.includes("Uni"))/* This will get you a list of only graduates who have "Uni" in the university */
.map(graduateFromUniversity => graduateFromUniversity.university) /* This will return an array of just the universities of the `filter` array of graduates from university */
.sort() // Will alphabetically sort that `map` array of university names alphabetically.
```